prisma/migrations/20240710170832_add_date_timestamp_to_video_metadata/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
+ALTER TABLE "VideoMetadata" ADD COLUMN     "date" TIMESTAMP(3),
+ADD COLUMN     "timestamp" INTEGER;
```

Comment:
Column `date` and `timestamp` sound very similar in this table. I think the column you wanted to add is `duration`, as per the given data.

Reply:
`duration` column is already present in VideoMetadata so should I remove the `timestamp` column? Also I will change `date` column's data type to store just date in schema.prisma.

Reply:
I'm not sure about that, I just cloned the repo and was trying to understand the codebase when I saw your PR. I hope someone from the frequent contributors can shed more light on this

Reply:
Okay, I will go ahead with other changes then